### PR TITLE
Datepickernullandux

### PR DESCRIPTION
**Pull request by sneridagh**

### CODE REVIEW

File: src/components/manage/Widgets/DatetimeWidget.jsx [hunk 1]
```diff
@@ -73,22 +86,26 @@ class DatetimeWidget extends Component {
...
+        ? // Since we assume UTC everywhere, then transform to local (momentjs default)
+          moment(this.props.value)
+        : // This might happen in old Plone versions dates
+          moment(`${this.props.value}Z`);
     }
 
-    if (!this.props.value && this.props.dateOnly) {
-      datetime.set(defaultTimeDateOnly);
-    }
+    // @nzambello do we need this if using null by default?
```
Comment (sneridagh) [inline src/components/manage/Widgets/DatetimeWidget.jsx:100]: @nzambello why was this here? Any use case?

File: src/components/manage/Widgets/DatetimeWidget.jsx [hunk 2]
```diff
@@ -73,22 +86,26 @@ class DatetimeWidget extends Component {
...
-    let datetime = moment().utc();
+    let datetime = null;
 
     if (this.props.value) {
       // check if datetime has timezone, otherwise assumes it's UTC
       datetime = this.props.value.match(/T(.)*(-|\+|Z)/g)
-        ? moment(this.props.value).utc()
-        : moment(`${this.props.value}Z`).utc();
+        ? // Since we assume UTC everywhere, then transform to local (momentjs default)
+          moment(this.props.value)
```
Comment (sneridagh) [inline src/components/manage/Widgets/DatetimeWidget.jsx:95]: Default for moment is to use local time, so let's do it!

File: src/components/manage/Widgets/DatetimeWidget.jsx [hunk 3]
```diff
@@ -141,9 +171,20 @@ class DatetimeWidget extends Component {
    * @returns {undefined}
    */
   onDateTimeChange = () => {
+    // Serialize always to UTC (toISOString)
     this.props.onChange(this.props.id, this.state.datetime.toISOString());
```
Comment (sneridagh) [inline src/components/manage/Widgets/DatetimeWidget.jsx:175]: We always serialize to UTC, using this method, all good.

File: src/components/manage/Widgets/DatetimeWidget.jsx (hunk 1 above)
Comment (nzambello) [reply src/components/manage/Widgets/DatetimeWidget.jsx:100]: If we are using null as default it's not necessary

File: src/components/manage/Widgets/DatetimeWidget.jsx [hunk 4]
```diff
@@ -73,22 +86,26 @@ class DatetimeWidget extends Component {
...
+    // @nzambello do we need this if using null by default?
+    // if (!this.props.value && this.props.dateOnly) {
+    //   datetime.set(defaultTimeDateOnly);
+    // }
 
     this.state = {
       focused: false,
-      isDefault: datetime.toISOString() === moment().utc().toISOString(),
+      isDefault:
+        datetime && datetime.toISOString() === moment().utc().toISOString(),
```
Comment (nzambello) [inline src/components/manage/Widgets/DatetimeWidget.jsx:108]:
```suggestion
        datetime?.toISOString() === moment().utc().toISOString(),
```
